sys/cddl/dev/systrace/systrace.c

```diff
@@ -112,7 +112,14 @@ extern const char *freebsd32_syscallnames[];
  * using a generated function. See sys/kern/makesyscalls.sh.
  */
 #include <sys/syscall.h>
+
+#if __has_feature(capabilities)
+#include <compat/cheriabi/cheriabi_proto.h>
+#include <compat/cheriabi/cheriabi_systrace_args.c>
+#else
```

Comment:
I don't see a corresponding change to the code to generate this file so this won't persist past a change to syscalls.master.

Reply:
This should change to have a FREEBSD64_SYSTRACE branch like FREEBSD32_SYSTRACE and then use the native ABI here, rather than bringing in the old CheriABI compat layer.

Reply:
will do that in another pr